src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -74,7 +73,7 @@ public async Task<(List<Error> errors, ContributionInfo contributionInfo)> GetCo
                         continue;
 
                     var contributor = await GetContributor(commit);
-                    if (contributor != null && !excludes.Contains(contributor.Name) && userIds.Add(contributor.Id))
+                    if (contributor != null && !excludes.Contains(contributor.Name))
```

Comment:
Add a debug assertion in line 79 `Log(login)` and line 114 `Log(email)` to ensure a single `login` or `email` is called at most once to prevent bugs like this:

![image](https://user-images.githubusercontent.com/511355/61762254-f9c3bb80-ae03-11e9-9f0d-6a7ac91a976f.png)